app/views/admin/users/index.html.haml

```diff
@@ -7,7 +7,7 @@
       %h1 Admin Page
       %h3 Users List
   %div.user_search
-    = form_tag admin_search_user_path do
+    = form_tag search_admin_users_path do
```

Comment:
Why was the path changed to search admin users? When you do rake routes it is admin_search_user.